web/src/features/fba/components/map/FBAMap.tsx

```diff
@@ -353,6 +364,36 @@ const FBAMap = (props: FBAMapProps) => {
           </Box>
         </Box>
       </MapContext.Provider>
+      <Box
+        sx={{
+          backgroundColor: '#FFFFFF',
+          border: '1px solid #000000',
+          padding: '12px',
+          position: 'absolute',
+          right: '8px',
+          bottom: '8px',
+          overflow: 'hidden'
+        }}
+      >
+        <Box
+          id="scalebar"
+          sx={{
+            ['& div.ol-scale-bar']: {
+              position: 'static'
+            },
+            ['& div.ol-scale-bar-inner div.ol-scale-step-marker']: {
+              top: '13px !important'
+            },
+            ['& div.ol-scale-bar-inner div>div.ol-scale-step-marker']: {
+              top: '-8px !important'
+            },
+            ['& div.ol-scale-step-text']: {
+              bottom: '5px !important'
+            }
+          }}
+          ref={scaleRef}
+        ></Box>
+      </Box>
```

Comment:
I tried creating a component out of this but got blocked with ref forwarding and integration with the OpenLayers' ScaleLine control which needs to have its target set to a ref.